dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Planner/OpenAIPlanner.cs

```diff
@@ -323,6 +323,18 @@ private string _GetModel(PromptTemplate promptTemplate)
             }
         }
 
+        /// <inheritdoc/>
+        public async Task<Plan> BeginTaskAsync(ITurnContext turnContext, TState turnState, AI<TState> ai, CancellationToken cancellationToken)
+        {
+            return await ContinueTaskAsync(turnContext, turnState, ai, cancellationToken);
+        }
+
+        /// <inheritdoc/>
+        public async Task<Plan> ContinueTaskAsync(ITurnContext turnContext, TState turnState, AI<TState> ai, CancellationToken cancellationToken)
+        {
```

Comment:
Parameter null check?
Better to add null check to all publicly exposed APIs.

Reply:
Updated.